### PR TITLE
LPS-202121: Add test cases for import Object Entries in CSV

### PR DESCRIPTION
**Pull request by sergiojimcos**
**What is new?**
- Add `decode` method to decode CSV string values given an enclosure parameter.
- Add test cases for import object entries in CSV.
- Refactor test cases

**Note**: The original PR starts from this [commit](cb8f39eecaed83f0241054d839d8608bb4d2f2fe) 

**JIRA Related Ticket**
https://liferay.atlassian.net/browse/LPS-202121

### CODE REVIEW

**Comment by sergiojimcos** [comment]
ci:test:relevant

**Review by 4lejandrito — CHANGES_REQUESTED** [review]
Also the `_assertEqualsExportCSV` method is broken because now a record can have more than one line. We need to properly parse the csv with csvparser and check record by record.

File: portal-kernel/src/com/liferay/portal/kernel/util/CSVUtil.java [hunk 1]
```diff
@@ -13,6 +13,29 @@
  */
 public class CSVUtil {
 
+	public static String decode(String enclosingChar, String s) {
+		if (s == null) {
+			return null;
+		}
+
+		if ((s.indexOf(CharPool.COMMA) < 0) &&
```
[the remark this answers is not in the document]
Comment (sergiojimcos) [reply portal-kernel/src/com/liferay/portal/kernel/util/CSVUtil.java:21]: What happen, if any user has not chosen a `enclosingChar` or `delimiter`?

[the remark this answers is not in the document]
Comment (4lejandrito) [reply portal-kernel/src/com/liferay/portal/kernel/util/CSVUtil.java:21]: Use `,` and `"` as default values.

**Comment by sergiojimcos** [comment]
ci:test:sf

**Comment by sergiojimcos** [comment]
ci:test:relevant

**Comment by sergiojimcos** [comment]
ci:test:sf

**Comment by sergiojimcos** [comment]
ci:test:relevant

**Comment by 4lejandrito** [comment]
@sergiojimcos see my changes.

**Comment by 4lejandrito** [comment]
ci:forward

**Comment by 4lejandrito** [comment]
ci:forward:force

**Pull request closed without merging**